### ISSUE TITLE
[Dependency version] click

### ISSUE THREAD

**Issue by yermulnik** [janeklb/gh-search#51]
Are there any plans to bump `click` version requirement?

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
gh-search 0.7.2 requires click~=7.1.2, but you have click 8.0.3 which is incompatible.
```

**Comment by janeklb** [comment]
I was not aware of the issue. Thanks for reporting it 👍 

will have a look at it soon

**Comment by yermulnik** [comment]
Thank you!

**Comment by janeklb** [comment]
Published 0.7.3: https://github.com/janeklb/gh-search/releases/tag/0.7.3

Should be up in pypi any second now

**Comment by yermulnik** [comment]
Awesome! Thank you!
